app/views/admin/participants/statuses/show.html.erb

```diff
@@ -129,6 +129,12 @@ end %>
   sl.with_row do |row|
     row.with_key(text: "induction status")
     row.with_value(text: govuk_tag(text: @participant_presenter.relevant_induction_record&.induction_status || "No Induction Record Found", colour: "grey"))
+    if policy(@participant_presenter.participant_profile).edit_induction_status? && ['withdrawn', 'leaving'].include?(@participant_presenter.relevant_induction_record&.induction_status)
```

Comment:
it seems quite a big piece of logic that can be put in the participant presenter (or another class).
It would be good to replace this line with something like `@participant_presenter.can_change_induction_status?` (or something like that).